kratos/python_scripts/run_cpp_tests.py

```diff
@@ -1,3 +1,19 @@
-from KratosMultiphysics import *
-Tester.SetVerbosity(Tester.Verbosity.TESTS_OUTPUTS)
-Tester.RunAllTestCases()
+import KratosMultiphysics as KM
+
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+
+try:
+    import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+except ImportError as e:
+    KM.Logger.PrintWarning("StructuralMechanicsApplication is not available")
+try:
+    import KratosMultiphysics.FluidDynamicsApplication as FluidDynamicsApplication
+except ImportError as e:
+    KM.Logger.PrintWarning("FluidDynamicsApplication is not available")
+try:
+    import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+except ImportError as e:
+    KM.Logger.PrintWarning("ConvectionDiffusionApplication is not available")
+
+KM.Tester.SetVerbosity(KM.Tester.Verbosity.TESTS_OUTPUTS)
+KM.Tester.RunAllTestCases()
```

Comment:
yikes
I would leave it as is, this causes more problems than it solves I think

@roigcarlo what do you say?

Reply:
In that case the tests depending on other applications will fail

Reply:
they shouldn't fail, they should be skipped

Reply:
Yes, technically skipo, but what I mean is that in the CI those test will not be run, therefore meaningless

Reply:
The CI importants every app and then rund all the tests

Reply:
*imports

Reply:
OK, i trust you